scalableminds/webknossos
# Being dataset uploader does not make editable true

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Create dataset with read acess by non-admin user.
- Set uploader in db.
- Ensure that the user still can not edit it (e.g. via PATCH /datasets/org/dataset/updatePartial)

Uploading a dataset requires managing permissions in that folder (see #7199 for an issue arising from that). That way the initial uploading logic should be unaffected.

### Issues:
- fixes #6780 

Also "uploader" is not used for remote datasets?

------
(Please delete unneeded items, merge only when none are left open)
